src/components/AudioTab/VersionDlg.tsx

```diff
@@ -41,28 +31,23 @@ export const VersionDlg = (props: IProps) => {
   const passages = useOrbitData<Passage[]>('passage');
 
   const [plan] = useGlobal('plan');
-  const [project] = useGlobal('project');
   const [memory] = useGlobal('memory');
-  const [offline] = useGlobal('offline');
-  const [offlineOnly] = useGlobal('offlineOnly');
   const { getPlan } = usePlan();
   const [user] = useGlobal('user');
   const [planRec] = useState(getPlan(plan) || ({} as Plan));
   const [playItem, setPlayItem] = useState('');
   const [data, setData] = useState<IRow[]>([]);
-  const [sectionArr, setSectionArr] = useState<[number, string][]>([]);
+  //const [sectionArr, setSectionArr] = useState<[number, string][]>([]);
   const [sectionMap, setSectionMap] = useState(new Map<number, string>());
-  const [shared, setShared] = useState(false);
   const [versions, setVersions] = useState<number[]>([]);
   const [refresh, setRefresh] = useState(0);
   const { IsVernacularMedia } = useArtifactType();
   const handleRefresh = () => setRefresh(refresh + 1);
   const allBookData: BookName[] = useSelector(
     (state: IState) => state.books.bookData
   );
-  const { getProjectDefault } = useProjectDefaults();
-  const { userIsAdmin } = useRole();
-  const [readonly] = useState((offline && !offlineOnly) || !userIsAdmin);
+
+  const { sectionArr } = useContext(PlanContext).state;
```

Comment:
This line won't work because VersionDlg is not only called from within the PlanContext. The reason it was coded like it was is so that it can be called from PlanContext or PassageDetailContext. (It is called twice from each place).